contrib/IECoreArnold/src/IECoreArnold/ShapeAlgo.cpp

```diff
@@ -176,6 +176,17 @@ void convertRadius( const std::vector<const IECore::Primitive *> &samples, AtNod
 void convertPrimitiveVariable( const IECore::Primitive *primitive, const PrimitiveVariable &primitiveVariable, AtNode *shape, const char *name )
 {
 
+	// make sure the primitive variable doesn't clash with built-ins
+	const AtNodeEntry *entry = AiNodeGetNodeEntry( shape );
+	if ( AiNodeEntryLookUpParameter(	entry, name ) != NULL ){
+		msg(
+			Msg::Warning,
+			"ToArnoldShapeConverter::convertPrimitiveVariable",
```

Comment:
`ToArnoldShapeConverter` isn't a think any more (although it once was). I suspect there are lingering references to it in other messages, and you picked this up from one of them? Can we replace with `ShapeAlgo::convertPrimitiveVariable`?